### DISCUSSION TITLE
catch SVT stream with android app FFmpeg?

### DISCUSSION THREAD

**Discussion by zticky** [stefansundin/privatkopiera#95] [Q&A]
Is it possible to catch SVT stream with the android app FFmpeg?
How should I change PC FFmpeg codec in Privatkopiera to make it work in the android app FFmpeg?

**Comment by Bub0** [comment]
ffmpeg in Android isn't the best but it can useful if you want to configure it manually.
It will error if you just copy the command from the addon but you can still download if you paste the m3u8 link(one input), choose output, choose copy in audio and video codec
It should look like,
-c:a copy -c:v copy
Or similar.
But the more easier way in Android is to use Lj video Downloader or 1dm+
Both can extract HLS and DASH and download the same ofc. You can also paste the HLS link from addon to the apps and download
Good luck! 🤞
Best regards

**Comment by zticky** [comment]
Thanx Bub0!
I been able to download audio and video stream with Lj video Downloader. Subtitle file was tricky, but found ADM app to download text file.
Next question.

How do I merge audio and video file on android?

**Reply by Bub0** [reply]
Lj muxes the A/V automatically if you don't want them to be separated otherwise I would recommend you to mux them with Ffmpeg. Make sure you use copy tho! Subtitle can be downloaded with 1DM+ too, even in HLS segments.
Good luck! :-)

**Reply by zticky** [reply]
How do I download two m3u8 linked files to one file in Lj video Downloader?
I don't see an option to set two m3u8 links.

**Reply by Bub0** [reply]
> How do I download two m3u8 linked files to one file in Lj video Downloader?
> I don't see an option to set two m3u8 links.

Choose hls-ts-avc.m3u8 in Privatkopiera at "Ström" then only copy the m3u8 link, and it should be muxed if you paste the link in the apps and download. It have been a long time since I used my phone to download so I can be wrong..
It should be the same as if you were using a, master.m3u8 with hls-ts-avc.m3u8. 
But if you use the extract from website in Lj, you don't need to use Privatkopiera at all. You can also use the browser in 1DM+ to fetch the streams without Privatkopiera! 
Sorry I didn't get you before. 
Best regards 👍

**Reply by zticky** [reply]
Thanx again Bub0!
You taught me a lot today.
Downloaded successfully merged video/audio file with Lj video Downloader.
